project/civiwiki/settings.py

```diff
@@ -167,23 +169,16 @@ def get_env_variable(environment_variable, optional=False):
 
 
 # Database
-if 'CIVIWIKI_LOCAL_NAME' not in os.environ:
-    STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
-
-    DATABASES = {
-        'default': dj_database_url.parse(get_env_variable("DATABASE_URL"))
-    }
-else:
-    DATABASES = {
-        'default': {
-            'HOST': 'localhost',
-            'PORT': '5432',
-            'NAME': get_env_variable("CIVIWIKI_LOCAL_NAME"),
-            'ENGINE': 'django.db.backends.postgresql_psycopg2',
-            'USER': get_env_variable("CIVIWIKI_LOCAL_USERNAME"),
-            'PASSWORD': get_env_variable("CIVIWIKI_LOCAL_PASSWORD"),
-        },
-    }
+DATABASES = {
```

Comment:
Use the **environment variables** to store your development database configuration. That way this code stays generic to development and production environments.

Also, be careful about removing the STATICFILES_STORAGE variable, as it may be important for production.